config/sync/core.entity_form_display.node.stanford_page.default.yml

```diff
@@ -84,6 +84,7 @@ content:
         stanford_wysiwyg: 3
         stanford_media_caption: 3
         stanford_spacer: 1
+        su_gallery: 12
```

Comment:
New naming convention?

Reply:
Thanks. that was left over from a previous attempt and then i fixed it https://github.com/SU-SWS/stanford_profile/pull/347/files#diff-a30e79017e2fc616c97d3ff9e2bca01ffe884b4508ace754be4d2345c62190f4R5